superset/exceptions.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+
+class NoDataException(Exception):
```

Comment:
This should derive `superset/utils/SupersetException`, we should have all exception in the same module, whether it's still utils or moving it to exceptions.py works for me as well.